tasks/manage_configs.yml

```diff
@@ -5,10 +5,12 @@
     src: celery_env.j2
     dest: "{{celery_env_dir}}/{{item.name}}"
   with_items: "{{celery_services}}"
+  notify: "restart celery services"
```

Comment:
This role isn't designed to start stop handlers itself. It registers them but the start and stop of service units is done outside of this role. The prime reason for this is that we create the service units before the source is checked out on the host. If you restart the services at that point, services will fail. One example would be on bootstrap of a new host. 

If services need to be restarted, you will need to have the role register a change that can be used in the playbook to call the appropriate handlers.